nhost/utils.go

```diff
@@ -18,6 +19,10 @@ const (
 	projectNameFile = "project_name"
 )
 
+var (
+	projectNameIgnoreRegex = regexp.MustCompile(`([^a-z0-9-])+`)
```

Comment:
```suggestion
	projectNameIgnoreRegex = regexp.MustCompile(`([a-z0-9-_])+`)
```

- I removed the `^` because I don't think we need it. But maybe we do. When using that character in your regex, did you have something specific?
- Added `_` as an allowed character.

Reply:
that regex says allow everything BESIDES alphanumeric and `-`

your regex will remove all alphanumeric, `-` and `_` 😄

Reply:
ah, okay. Then can we just update to also allow `_`? Do you think that makes sense?

Reply:
yeah, I think so. Will update + adjust tests

Reply:
I'll trim `_` in the beginning and in the end of the name, they aren't allowed to be there by docker compose